src/main/java/org/cryptomator/cli/pwd/PasswordFromStdInputStrategy.java

```diff
@@ -0,0 +1,52 @@
+package org.cryptomator.cli.pwd;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.Console;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class PasswordFromStdInputStrategy implements PasswordStrategy {
+    private static final Logger LOG = LoggerFactory.getLogger(PasswordFromStdInputStrategy.class);
+
+    private final String vaultName;
+
+    public PasswordFromStdInputStrategy(final String vaultName) {
+        this.vaultName = vaultName;
+    }
+
+    @Override
+    public String password() {
+        LOG.info("Vault " + "'" + vaultName + "'" + " password from standard input.");
+
+        String password = "";
+        Console console = System.console();
+        if (console == null) {
+            LOG.warn("No console: non-interactive mode, instead use insecure replacement, PW is shown!");
+
+            BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+            LOG.info("Enter password: ");
```

Comment:
Printing the prompt through a `LOG` call only works well if logging is done to stdout. Maybe we should explicitly use stdout / the console to prompt for the password? Not a problem as long as we do logging the way we do it now. So must not be adressed to merge this.

Reply:
You are right, I will change this.